### PR TITLE
add aiida-lammps recipe

### PR DESCRIPTION
**Pull request by chrisjsewell**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/aiida-lammps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{% set name = "aiida-lammps" %}
+{% set version = "0.4.0b3" %}
```
Comment (scopatz) [inline recipes/aiida-lammps/meta.yaml]: Is there a non-beta / alpha version that can be released instead?

Comment (chrisjsewell) [reply recipes/aiida-lammps/meta.yaml]: Hey @scopatz, not yet I'm afraid. The package is written around `aiida-core` v1.0.0, which is on beta-3 release. The alpha release should be out soon hopefully.
Note the tests should be fixed once conda-forge/aiida-core-feedstock#15 is merged

Comment (chrisjsewell) [reply recipes/aiida-lammps/meta.yaml]: @scopatz the tests are now passing

Comment (scopatz) [reply recipes/aiida-lammps/meta.yaml]: We generally don't package beta or alpha releases. How soon is soon?

Comment (chrisjsewell) [reply recipes/aiida-lammps/meta.yaml]: The final beta (b4) was released yesterday, and the stable release is scheduled for end of july

Comment (scopatz) [reply recipes/aiida-lammps/meta.yaml]: Well is there not something in the 0.3 series?

**Pull request closed without merging**